src/app/pages/MediaAssetPage/fixtureData.json

```
@@ -1860,7 +1860,7 @@
   },
   "promo": {
     "headlines": {
-      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < >",
+      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh, bye bye Khoa & < >",
```

```suggestion
      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >",
```
trolling aside 😅 thanks for the love my Scottish friends 💪